Neuron/Services/Ethereum/EthereumNetwork.swift

```diff
@@ -44,6 +44,18 @@ struct EthereumNetwork {
         case kovan
 
         static let allValues = allCases.map { $0.rawValue }
+        var chainName: String {
+            switch self {
+            case .mainnet:
+                return "Ethereum Mainnet"
+            case .rinkeby:
+                return "Ethereum Rinkebynet"
+            case .ropsten:
+                return "Ethereum Ropstennet"
```

Comment:
I don't think anybody call `Rinkeby` `Rinkebynet`, same for `Ropsten` and `Kovan`.